.pipelines/pipeline.user.linux.official.all_tag.all_phase.all_config.ci_prod.yml

```diff
@@ -28,6 +28,19 @@ build:
       name: 'Build Docker Provider Shell Bundle'
       command: '.pipelines/build-linux.sh'
       fail_on_stderr: false
+      artifacts:
+        - from: 'deployment'
+          to: 'build'
+          include:
+            - '**'
+        - from: '.pipelines'
+          to: 'build'
+          include:
+            - '**'
+        - from: 'kubernetes'
+          to: 'build'
+          include:
+            - 'omsagent.yaml'
```

Comment:
here, can we copy the all '*.yaml' so we can deploy the configmap if required for any of the scenario?